project/ROADMAP.md

```diff
@@ -0,0 +1,5 @@
+Galaxy Project Roadmap
```

Comment:
So this file would disappear and its contents would be recorded similar to  https://github.com/nodejs/node/issues/2522 (using issues and milestones hyperlinking). This is similar to what @jmchilton and @jxtx mentioned above, just more poorly worded.

Also see another node example of meta-planning: https://github.com/nodejs/node/issues/3000